app/main.py

```diff
@@ -1,3 +1,54 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: (int, float)) -> None:
```

Comment:
gj, but use annotations like this: `int | float` or `Union[int, float]`

Reply:
fixed